### DISCUSSION TITLE
Neovim user. Can I do `s/foo/bar/g` in zed editor?

### DISCUSSION THREAD

**Discussion by mhemken-vts** [zed-industries/zed#10815] [Feature Requests]
This is something I use a lot. I want to change the name of a variable. So I use the sed like thing in neovim, eg. `s/foo/bar/g`.

This doesn't work in zed out of the box.

Is it supported? If so how do I do this (in vim it's just `:`)?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
